### PR TITLE
Typos + thumbnail hover glitch

### PR DESCRIPTION
**Pull request by gilren**
Fixed some typos/spelling errors on multiple files,

https://github.com/bdebon/youtube-thumbnail-tester-chrome-extension/commit/1874af4564abb0f46d6defc8bc9cca0afe96e7da Forces the thumbnail to take all the available space so that we can't see the cat's paw on hover.
Added a border based on https://github.com/bdebon/youtube-thumbnail-tester-chrome-extension/issues/27

Might wanna make a check for a min size of the file to prevent stretching it when it's smaller than the wanted size.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -341,11 +337,12 @@ input {
...
-    z-index: 10;
+    z-index: 6;
     display: none;
-    top: 50%;
-    transform: translateY(-50%);
-    max-width: 100%;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    border: 6px solid var(--btnBg);
```
Comment (bdebon) [inline style.css:345]: Why such a big border? Is it in the original design?

**Review by bdebon — COMMENTED** [review]
Perfect, thank you for fixing all the typos.
I just have one question about the big border to fix the paw cat glitch?

File: style.css (hunk 1 above)
Comment (gilren) [reply style.css:345]: Yeah so the border doesn't fix the glitch, I just added it preemptively as it was missing based on https://github.com/bdebon/youtube-thumbnail-tester-chrome-extension/issues/27

Comment (bdebon) [reply style.css:345]: Ah you're 100% right!
Then thank you for all the good work! I'm gonna proudly merge it!

**Pull request merged**